src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+package src.main.java;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    static int totalNumberOfTask = 0;
```

Comment:
Maybe you could add a destructor method that decrements totalNumberOfTask when the task is deleted?